_episodes/05-contribution-governance-communication.md

```diff
@@ -1,52 +1,380 @@
+redistribute responsibilities.
+
+A clearly defined governance structure not only allows your contributors to identify the correct
+contact person, but also helps them to identify roles that they can fill in your project.
+
+A properly managed project uses governance to make sure that there are no responsibilities
+left behind, unassigned, and optimizes for the rate of development progress.
+
+> ## Challenge
+> #### Time: 10 min
 >
-> Let's discuss what types of communication media do people prefer. Think about the team size, 
+> Assign levels of priority needed in the project design and governance
+> (choose one level per line and justify your choice)
+>
+> |---|---|---|---|---|
+> |Topic|High|Medium|Low|Justify your choice|
+> |About the project|||||
+> |Ownership|||||
+> |Roles and Responsibilities|||||
+> |License|||||
+> |Funding|||||
+> |Timeline|||||
+> |Outreach||||||
+>
+> > ## Solution
+> >
+> > Discussions among learners who chose different priorities will convince them of the need
+> > for governance and transparency in open source projects. This will clarify the contributors' roles
+> > and responsibilities within projects.
+> {: .solution}
+{: .challenge}
+
+## Communication
+
+Let's now talk about transparent communication processes. There are several options available
+to establish transparent communication with everyone involved in an open source project.
+However, the choice of communication channel differs depending on the team size, which also
+influences the frequency of communications. From the start of your project, use open
+and public communication channels for members of your project. Mailing lists and other online
+forums work well for open source projects, and scale well as your project grows. As your user base
+grows you may need to create additional lists only for important announcements for instance.
+
+> ## Discussion: Develop communication strategies
+> #### Time: 5 minutes
+> Let's discuss what types of communication media do people prefer. Think about the team size,
 > and what would be most convenient way for people to communicate and get updates.
-> You have 5 min to discuss, in groups.
+>
+> > ## Solution
+> >
+> > Here is a list of recommendations. However, the efficient solutions will
+> > depend on the frequency of communication and the team size.
+> > - Community calls/meetings
+> > - Email list
+> > - Slack
+> > - Twitter
+> > - GitHub issues
+> > - Announcements (newsletters/updates)
+> > - Archive
+> {: .solution}
 {: .discussion}
 
-> ## `.testimonial`
-> “For me this experience has been incredible! Thank you for putting all your effort into this 
-> project. Many people responded with overflowing enthusiasm to participate and collaborate in the 
-> different facets of this project. It was a pleasure to work with all of you.” - H.S.
+## Recommendations
+
+The following checklist is only recommended key points to consider in making contributions easier, clear and transparent.
```

Comment:
-> the following checklist contains some points to be addressed in order to make contributions easier, clearer and more transparent